test/integration/multitenant/multitenant_test.go

```diff
@@ -110,6 +110,10 @@ func TestMultitenant(t *testing.T) {
 					clusterName := regexp.MustCompile(`\/clusters\/([^\/]*)$`).FindStringSubmatch(membershipOp.Get("endpoint.gkeCluster.resourceLink").String())[1]
 					clusterOp := gcloud.Runf(t, "container clusters describe %s --location %s --project %s", clusterName, clusterLocation, clusterProjectID)
 
+					// Extract enablePrivateEndpoint flag value
+					enablePrivateEndpoint := clusterOp.Get("privateClusterConfig.enablePrivateEndpoint").Bool()
```

Comment:
I believe the public endpoint IP is still assigned, just can't be used.  Instead I'd suggest checking `privateClusterConfig.enablePrivateEndpoint = true` on the cluster.

Reply:
you are right, the external ip still exists but its disabled, I changed the test to check the `enablePrivateEndpoint` field.